js/jsscriptfiles_set2.js

```diff
@@ -0,0 +1,29 @@
+var numberArray = [0, 6, 4, 2, 9, 44, 23, 12];
+//console.log(numberArray);
+
+//Function to take the array as an input to take it for sorting
+function printTheNumberArray(){
+	
+	// Condition to check if the array is empty
+	if (numberArray.length == 0) {
+		console.error("Pls supply a valid array with elements");
+	}
+
+	// array is not empty, proceed with sorting the array
+	else if (numberArray.length != 0){
+		numberArray.sort(
```

Comment:
Smart move, but do not forget that I want the steps how often you swapped the numbers. This will not be possible with this method call, so you have to implement the sorting yourself.

Reply:
Thanks, but the sort was not working originally. I've to figure out, still not clear a bit.
And yeah, the algorithmic sorting is still pending. Hence, provided the comment in PR.